edx_repo_tools/oep2/explode_repos_yaml.py

```diff
@@ -8,7 +8,7 @@
 import yaml
 
 from edx_repo_tools.auth import pass_github
-from edx_repo_tools.data import iter_openedx_yaml
+from edx_repo_tools.data import iter_openedx_release_yaml
```

Comment:
You no longer renamed this function but you did change the function signature, I think you need to update calls to this function.